app/views/layouts/application.html.erb

```diff
@@ -3,30 +3,30 @@
 <head>
   <meta http-equiv="content-type" content="text/html; charset=utf-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
-  <%= favicon_link_tag asset_path('favicon.png') %>
-  <!--
-  <link rel="apple-touch-icon" sizes="57x57" href="/apple-touch-icon.png" />
-  <link rel="apple-touch-icon" sizes="114x114" href="/apple-touch-icon.png" />
-  <link rel="apple-touch-icon" sizes="72x72" href="/apple-touch-icon-144.png" />
-  <link rel="apple-touch-icon" sizes="144x144" href="/apple-touch-icon-144.png" />
-  -->
+
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
```

Comment:
Il file non c'è nel repo, già che ci sei aggiungilo per favore

Reply:
Il file c'è, lo trovi in /public

Reply:
Se faccio il checkout `develop` non mi spunta, neanche con `find . | grep apple`. Può essere che mi sbaglio io, però non lo trovo nè là nè dall'interfaccia web di gambero né con `grep -R apple`.

Reply:
done